manifests/install.pp

```diff
@@ -33,9 +33,9 @@
   if $cvmfs_cache_base != $default_cvmfs_cache_base {
     file { $cvmfs_cache_base:
       ensure  => directory,
-      owner   => cvmfs,
-      group   => cvmfs,
-      mode    => '0700',
+      owner   => $cvmfs_cache_owner,
```

Comment:
Would be good to have some tests.

Reply:
I've got to be doing something obvious wrong in the unit test I just wrote, but I don't see it...